limbo/slack.py

```diff
@@ -68,9 +43,6 @@ def __init__(self, token):
         self.connected = False
         self.pingcounter = 0
 
-    def api_call(self, method, **kwargs):
```

Comment:
wow - i'm honestly surprised that python wouldn't error on naming two functions identically. not even a pylint warning! :sweat_smile:

Reply:
pylint does actually flag it (phew) haha, I just wasn't running lint :)